packages/flutter_tools/lib/src/commands/run.dart

```diff
@@ -109,6 +109,9 @@ class RunCommand extends RunCommandBase {
         hide: !verboseHelp,
         help: 'Turn on strong mode semantics.\n'
               'Valid only when --preview-dart-2 is also specified');
+    argParser.addFlag('track-widget-creation',
+        hide: !verboseHelp,
+        help: 'Track widget creation locations. Requires Dart 2.0 functionality.');
```

Comment:
no default value?

Reply:
The default default value for flags is false